README.md

```diff
@@ -89,11 +89,13 @@ There are ongoing efforts to support parts of it to make at least some popular m
 ## Build Instructions
 ### Linux
 Install dependencies:
-* Ubuntu 20.04
+* Ubuntu 20.04/22.04
 ```bash
+# obtain the VERSION_CODENAME
+source /etc/os-release
```

Comment:
I could avoid the comment by making the code self-explanatory while also avoiding sourcing of the unnecessary variables, if you'd prefer this approach:
`source <(cat /etc/os-release | grep VERSION_CODENAME)`

Reply:
Hi, @Nindaleth and thanks for looking into this!

Yes, ideally build instruction should be copy-paste friendly. Sad that it getting more and more complex with `VERSION_CODENAME`

Reply:
OK, change is done.

With yum/dnf you can create a single repo using special variables such as `baseurl=http://linux.dropbox.com/fedora/$releasever/` that will always pick the appropate packages, not sure if that's possible with apt but LunarG would make this easier if they used it. But that's not a thing we can change from here :-)